docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc

```diff
@@ -16,6 +16,6 @@ CALL apoc.schema.relationships();
 .Results
 [opts="header"]
 |===
-| name                                                         | type                              | properties | status
-| "CONSTRAINT ON ()-[liked:LIKED]-() ASSERT exists(liked.day)" | "RELATIONSHIP_PROPERTY_EXISTENCE" | ["day"]    | ""
+| name                                                             | type                              | properties | status
+| "CONSTRAINT ON ()-[liked:LIKED]-() ASSERT liked.day IS NOT NULL" | "RELATIONSHIP_PROPERTY_EXISTENCE" | ["day"]    | ""
```

Comment:
Actually this should be updated to use `FOR .. REQUIRE` as well

Reply:
Yes. I get over 100 matches on `CONSTRAINT ON` on apoc dev though so I think we need a separate PR for that though if you don't have one open already?

Reply:
Sure lets do it separately then :)